src/modules/articlesDataModule.js

```diff
@@ -1,7 +1,7 @@
 import axios from 'axios'
 
-const getExploreArticles = async () => {
-  let response = await axios.get('/articles?article_type=story')
+const getExploreArticles = async (articleType) => {
```

Comment:
Also, why is this called `getExploreArticles` when you receive stories through the same function as well?